### PR TITLE
Commit SUclass changes from Ginkgo rollout

### PR DESCRIPTION
**Pull request by stvstnfrd**
These were changes I had made for the ginkgo rollout, but hadn't landed in master.

### CODE REVIEW

File: stanford/suclass/ansible.cfg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [defaults]
 callback_whitelist = profile_tasks
 host_key_checking = False
-inventory = ../../playbooks/edx-east/ec2.py
+inventory = inventory.yaml
```
Comment (caesar2164) [inline stanford/suclass/ansible.cfg]: this seems reasonable

Comment (stvstnfrd) [reply stanford/suclass/ansible.cfg]: It's not just "reasonable"; it's what we're doing everywhere :)

File: playbooks/roles/rabbitmq/templates/rabbitmq-env.conf.j2 [hunk 2]
```diff
@@ -1,3 +1,3 @@
 RABBITMQ_NODE_PORT={{ rabbitmq_port }}
 RABBITMQ_NODE_IP_ADDRESS={{ rabbitmq_ip }}
-RABBITMQ_USE_LONGNAME={{ RABBITMQ_USE_LONGNAME }}
+USE_LONGNAME={{ RABBITMQ_USE_LONGNAME }}
```
[the remark this answers is not in the document]
Comment (stvstnfrd) [reply playbooks/roles/rabbitmq/templates/rabbitmq-env.conf.j2]: Those aren't variables; it's a config file...

[the remark this answers is not in the document]
Comment (caesar2164) [reply playbooks/roles/rabbitmq/templates/rabbitmq-env.conf.j2]: right, but so this means the config vars name changed?

**Pull request merged**